draft-ietf-quic-recovery.md

```diff
@@ -1259,6 +1259,9 @@ SetLossDetectionTimer():
 
   sent_time, _ = GetEarliestTimeAndSpace(
     time_of_last_sent_ack_eliciting_packet)
+  if (sent_time == 0)
+    assert(!PeerNotAwaitingAddressValidation())
+    sent_time = now()
   loss_detection_timer.update(sent_time + timeout)
```

Comment:
```suggestion
  if (sent_time != 0)
    loss_detection_timer.update(sent_time + timeout)
```

Reply:
We need to arm the timer for anti-deadlock reasons even if sent_time is 0, so I think this needs to stay as is?

Reply:
My bad.  I didn't think that through.  Would it instead be better to change this line:

```
  time_of_last_sent_ack_eliciting_packet[kPacketNumberSpace] = 0
```
to 
```
  time_of_last_sent_ack_eliciting_packet[kPacketNumberSpace] = now()
```

Reply:
I'm worried that's too clever.  This is a special case, but so are many things about the anti-deadlock functionality, unfortunately.